deploy/.env.example

```diff
@@ -5,6 +5,10 @@ DEPLOYMENT_GAS_LIMIT=4000000
 DEPLOYMENT_GAS_PRICE=10
 GET_RECEIPT_INTERVAL_IN_MILLISECONDS=3000
 
+BRIDGEABLE_TOKEN_NAME="Your New Bridged Token"
```

Comment:
As per our discussion on the call please consider to
1. make a configurable option to use already deployed token on Home side (`erc20-to-erc20` mode, https://github.com/poanetwork/poa-bridge-contracts/issues/70)
2. have ability to deploy an erc20/677 token on Foreign side (`erc20-to-erc20` mode)

In this case we will need the following configuration parameters:
```
HOME_ERC677_TOKEN_ADDRESS 
HOME_ERC677_TOKEN_NAME
HOME_ERC677_TOKEN_SYMBOL
HOME_ERC677_TOKEN_DECIMALS
```
and 
```
FOREIGN_ERC20_TOKEN_ADDRESS 
FOREIGN_ERC677_TOKEN_NAME
FOREIGN_ERC677_TOKEN_SYMBOL
FOREIGN_ERC677_TOKEN_DECIMALS
```

where `FOREIGN_ERC20_TOKEN_ADDRESS` will be used instead of `ERC20_TOKEN_ADDRESS`.